assets/page/SettingsAccount.svelte

```diff
@@ -9,19 +9,36 @@ import {getContext} from 'svelte';
 import {l} from '../js/i18n';
 import {route} from '../store/Route';
 
-const colorSchemeOptions = [['auto', 'Auto'], ['dark', 'Dark'], ['light', 'Light']];
 const user = getContext('user');
 const updateUserOp = user.api.operation('updateUser');
 
+// "colorSchemeOptions" will get calculated each time "theme" changes
```

Comment:
We don't really need this comment, it was just me trying to explain what this line does.

Reply:
Removed